models/perpetual/perpetual_trades_legacy.sql

```diff
@@ -1,9 +1,59 @@
 {{ config(
 	tags=['legacy'],
 	
-        alias = alias('trades', legacy_model=True)
+        alias = alias('trades', legacy_model=True),
+        partition_by = ['block_date'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+	      unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index'],
+        post_hook='{{ expose_spells(\'["optimism","avalanche_c","arbitrum"]\',
+                                "sector",
+                                "perpetual",
+                                \'["msilb7", "drethereum", "rplust","Henrystats", "jeff-dude"]\') }}'
         )
 }}
 
-SELECT 
-1 as dummy 
\ No newline at end of file
+{% set perpetual_trade_models = [
+ ref('perpetual_protocol_perpetual_trades_legacy')
+,ref('pika_perpetual_trades_legacy')
+,ref('synthetix_perpetual_trades_legacy')
+,ref('emdx_avalanche_c_perpetual_trades_legacy')
+,ref('hubble_exchange_avalanche_c_perpetual_trades_legacy')
+,ref('gmx_perpetual_trades_legacy')
```

Comment:
if you aren't already actively looking, these would all need restored too

Reply:
sorry I don't understand what you mean by restored

Reply:
I didn't edit the legacy models of the projects, I originally did but then I reverted them, before they got reviewed/merged.

Reply:
only changes I made was to  perpetual_trades_legacy.sql

Reply:
ah, got it, sounds good. i will do final check and merge again in the morning 🤝

Reply:
thanks ser!